contracts/BIFI/strategies/Sonne/StrategyLendingSonne.sol

```diff
@@ -0,0 +1,407 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin-4/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin-4/contracts/token/ERC20/utils/SafeERC20.sol";
+
+import "../../interfaces/common/IUniswapRouterETH.sol";
+import "../../interfaces/common/IComptroller.sol";
+import "../../interfaces/common/IVToken.sol";
+import "../Common/StratFeeManager.sol";
+import "../../utils/GasFeeThrottler.sol";
+
+
+//Lending Strategy 
+contract StrategyLendingSonne is StratFeeManager, GasFeeThrottler {
+    using SafeERC20 for IERC20;
+
+    // Tokens used
+    address public native;
+    address public output;
+    address public want;
+    address public iToken;
+
+    // Third party contracts
+    address constant public comptroller = 0x60CF091cD3f50420d50fD7f707414d0DF4751C58;
```

Comment:
If we pass this in, is there anything else stopping this from being a generic "Balancer V2" strat rather than Sonne-specific?  If so can rename.

Reply:
Yep, I also like the idea! Went with StrategyCompoundV2